src/README.md

```diff
@@ -3,6 +3,115 @@
 
 - You may refer to [cros-diag-app/diagnostics-extension](https://github.com/MahmoudAGawad/cros-diag-app/tree/main/diagnostics-extension) to see the integration of this library in action.
 
+
+[comment]: <Contents below is manually generated from the source code.>
+Contents
+========
```

Comment:
I think we don't need this table of content as there is build-in version.

Context: https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/

Reply:
Thanks for sharing. Remove TOC.